### PR TITLE
Fixt tests

### PR DESCRIPTION
**Pull request by janinko**

### CODE REVIEW

**Comment by stanlyDoge** [comment]
retest this please

File: testsuite/src/test/java/org/jboss/da/test/client/rest/reports/RestApiReportsRemoteTest.java [hunk 1]
```diff
@@ -84,4 +88,14 @@ public void testReportWithoutDependencies() throws Exception {
...
+    @Override
+    protected void assertEqualsJson(String expected, String actual) {
+        try {
+            JSONAssert.assertEquals(expected, actual, JSONCompareMode.LENIENT);
+        } catch (JSONException ex) {
+            fail("The test wasn't able to compare JSON strings" + ex);
+        }
+    }
+
 }
```
Comment (stanlyDoge) [inline testsuite/src/test/java/org/jboss/da/test/client/rest/reports/RestApiReportsRemoteTest.java]: Could you add the same piece of code to WebsocketApiReportsTest? Thanks.

**Comment by janinko** [comment]
retest this please

**Comment by janinko** [comment]
retest this please

**Comment by janinko** [comment]
retest this please

**Pull request merged**